core/services/ocr2/plugins/ocr2keeper/evm21/upkeepstate/store.go

```diff
@@ -84,13 +109,17 @@ func (u *upkeepStateStore) Start(context.Context) error {
 
 	{
 		go func(ctx context.Context) {
-			ticker := time.NewTicker(GCInterval)
+			ticker := time.NewTicker(u.cleanCadence)
```

Comment:
```suggestion
			ticker := time.NewTicker(utils.WithJitter(u.cleanCadence))
```

Reply:
Shit. Missed this one again because of the merge commit. Thanks for flagging.